components/gantt/accessibility/overview.md

```diff
@@ -0,0 +1,113 @@
+
+# Blazor Gantt Accessibility Example
+
+WCAG 2.2 introduces the <a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements" target="_blank">"Dragging Movements"</a> criterion as an important part of the Operable principle. Its primary goal is to guarantee that any feature reliant on drag actions offers an alternative method that can be executed with a single click, enhancing user accessibility.
+
+The illustrative example below shows the Gantt tree columns reorder action, achievable through the [Column Menu]({%slug gantt-column-menu%}). Telerik UI for Blazor aims to offer a versatile API that allows users to trigger all functions programmatically or externally, meeting diverse accessibility requirements for any applications.
+
+The following example demonstrates the [accessibility compliance of the Gantt component]({%slug gantt-wai-aria-support%}). The described level of compliance is achievable with the [Ocean Blue A11y Accessibility Swatch]({%slug themes-accessibility-swatch%}).
+
+>caption Gantt accessibility compliance example
+
+````CSHTML
+@*Evaluate the example with Axe Core or other accessibility tools*@
+
+<TelerikGantt Data="@Data"
+              Width="900px"
+              Height="600px"
+              IdField="Id"
+              ParentIdField="ParentId"
+              Navigable="true"
+              ColumnReorderable="true"
+              Sortable="true"
+              ColumnResizable="true"
+              ShowColumnMenu="true">
+    <GanttViews>
+        <GanttDayView></GanttDayView>
+        <GanttWeekView></GanttWeekView>
+        <GanttMonthView></GanttMonthView>
+        <GanttYearView></GanttYearView>
+    </GanttViews>
+    <GanttColumns>
+        <GanttColumn Field="@nameof(FlatModel.Title)" ShowColumnMenu="false">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.PercentComplete)">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.Start)">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.End)">
+        </GanttColumn>
+    </GanttColumns>
+</TelerikGantt>
+
+@code {
+    private DateTime SelectedDate { get; set; } = new DateTime(2019, 11, 11, 6, 0, 0);
+
+    public class FlatModel
+    {
+        public int Id { get; set; }
+        public int? ParentId { get; set; }
+        public string Title { get; set; }
+        public double PercentComplete { get; set; }
+        public DateTime Start { get; set; }
+        public DateTime End { get; set; }
+    }
+
+    private int LastId { get; set; } = 1;
+    private List<FlatModel> Data { get; set; }
+
+    protected override void OnInitialized()
+    {
+        Data = new List<FlatModel>();
+        var random = new Random();
+
+        for (int i = 1; i < 6; i++)
+        {
+            var newItem = new FlatModel()
+                {
+                    Id = LastId,
```

Comment:
You can use `++LastId` for the Id assignment, instead of incrementing the variable manually separately. See https://docs.telerik.com/blazor-ui/knowledge-base/grid-inline-edit-form